andela/AndelaEatsUI
# ft(menuModal): add vendor engagement input field

**What does the PR do?**
- This PR adds `vendor engagement input field` in` Menu Modal`

**Description of Task to be completed?**
- add vendor engagement input field

**How should this be manually tested?**
- Clone the repository [here](https://github.com/andela/AndelaEatsUI.git)
-  cd into AndelaEatsUI and checkout to this branch `ft-add-vendor-input-on-menu-modal-161220355`
- run these commands `npm run bundle` and `npm run dev` 
- Log in as an `Admin`, navigate to `admin dashboard`, select `Menu Icon`, and click on `Add Menu`
- run the command `npm run test-dev` to ensure all tests are passing

**What are the relevant pivotal tracker stories?**
- [#161220355](https://www.pivotaltracker.com/story/show/161220355)

**Screenshots**
<img width="1679" alt="screen shot 2018-10-17 at 3 11 32 pm" src="https://user-images.githubusercontent.com/26305570/47092487-fa00ac00-d21e-11e8-8d4a-9b28e5770578.png">
<img width="1680" alt="screen shot 2018-10-17 at 3 12 18 pm" src="https://user-images.githubusercontent.com/26305570/47092539-1a306b00-d21f-11e8-8cc0-2f9f797a7e13.png">
